### ISSUE TITLE
64-bit support(?)

### ISSUE THREAD

**Issue by QuantumToasted** [sigsegv-mvm/TFMaxVoiceSpeakDelayUnlimiter#3]
Hi there! I am an avid fan of imploring upon my teammates my desire for a dispenser very frequently. I tried loading this extension in the normal way on my 64-bit server (linux) by dropping it in `sourcemod/extensions`, but then I receive a console error:

```
L 05/21/2024 - 17:16:47: [SM] Unable to load extension "TFMaxVoiceSpeakDelayUnlimiter.ext": /home/steam/tf-dedicated/tf/addons/sourcemod/extensions/x64/TFMaxVoiceSpeakDelayUnlimiter.ext.so: cannot open shared object file: No such file or directory
```

I am unsure if the solution is as simple as just copying the extension file to there like so, or if something special needs to be done I.E. actually modifying the extension. Whether this issue just ends up being me letting other 64-bit users know what needs to be done (copying the file), or if actual changes need to be made to the extension, either way works for me!

**Comment by sigsegv-mvm** [comment]
The extension would need to be ported for x64 and recompiled that way. You can't just rename the compiled binary file; that doesn't work.

Now, on the plus side, I think it likely wouldn't require a tremendous number of changes to the source code for this particular SM extension; but I believe it's still a bit more involved than just running the exact same source code through the compiler with different target architecture options.

I can maybe consider looking at what's required to port this to AMD64, but no guarantees.

**Comment by QuantumToasted** [comment]
> You can't just rename the compiled binary file; that doesn't work.

Yeah, I figured that out when I tried plopping it into the x64 folder as-is. Can't help just trying it to see, I guess?

I know this extension is likely not at the top of your priority list, but at least know that there's one person out there who greatly appreciates your contributions! I don't mind dropping back to a 32-bit server install since I know a lot of Sourcemod stuff is *still* broken on 64-bit servers (DHooks comes to mind), so if I need any of that functionality, I'll either need to be very, very patient, or just make the switch back. In either case, thank you once again!
